automatic/graphviz/tools/chocolateyUninstall.ps1

```diff
@@ -12,7 +12,22 @@ if ($key.Count -eq 1) {
             fileType               = 'EXE'
             validExitCodes         = @(0)
             file                   = "$($_.UninstallString.Replace(' /x86=0', ''))"   #"C:\Program Files\OpenSSH\uninstall.exe" /x86=0
+            softwareName           = 'Graphviz*'
         }
+
+        $installLocation = Get-AppInstallLocation $packageArgs.softwareName
+        Write-Debug "$packageName installed in: $installLocation"
+        
+        if (!$installLocation) { 
+            Write-Warning "Can't find $packageName install location"; return 1
```

Comment:
As with previous comment, please don't return `1` here.
Just return, without any exit code.

Reply:
Fixed. Thank you for the feedback!